ci/tasks/scripts/run.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
 if [ -n "${ECS_CONTAINER_METADATA_URI}" ]; then
-  export TASK_ID=$(curl $ECS_CONTAINER_METADATA_URI | jq -r '.Labels ."com.amazonaws.ecs.task-arn"')
+  FULL_TASK_ARN=$(curl $ECS_CONTAINER_METADATA_URI | jq -r '.Labels ."com.amazonaws.ecs.task-arn"')
```

Comment:
What about 
```
export TASK_ID=$(curl $ECS_CONTAINER_METADATA_URI | jq -r '.Labels ."com.amazonaws.ecs.task-arn"' |  sed 's/.*\///')
```

or you could use your solution and use 
```
| tr "/" "\n" | tail -n 1
```

Reply:
I tried to reproduce both versions (sed and tr) in a local bash script, using one of the currently displayed ARNs as a basis but got either a blank output or 'No such file or directory'. 

E.g.

ARN=$("arn:aws:ecs:eu-west-2:788375279931:task/wifi-frontend-cluster/275ddf4dd4734c9fbe80036d3826cdfe" | tr "/" "\n" | tail -n 1)

or..

ARN=$("arn:aws:ecs:eu-west-2:788375279931:task/wifi-frontend-cluster/275ddf4dd4734c9fbe80036d3826cdfe" |  sed 's/.*\///')

or either without the $().

Reply:
```ARN=$(echo "arn:aws:ecs:eu-west-2:788375279931:task/wifi-frontend-cluster/275ddf4dd4734c9fbe80036d3826cdfe" | sed 's/.*\///')```

works. I think you forgot the 'echo' in your example

Reply:
Thank you! I edited your comment above just to XXX out the account no.

Reply:
Updated and rebased to use @koetsier solution with tail/tr.